src/main/util.ts

```diff
@@ -31,4 +31,6 @@ export const AUTOAV_BACKGROUND_THREAD_PATH = app.isPackaged
 
 export const getResourcePath = (...paths: string[]): string => {
   return path.join(RESOURCES_PATH, ...paths);
-};
\ No newline at end of file
+};
+
+export const appdataPath = path.join(process.env.APPDATA ?? "C:\\Users\\Public", 'fimav-assistant');
```

Comment:
If we use `app.getPath('userData')`, then it'll work fine on any platform. Plus that's already where the config file lives

Reply:
fixed in fa5dfa8